### ISSUE TITLE
Alpine 3.7 bug, skalibs/posixplz.h: No such file or directory 

### ISSUE THREAD

**Issue by metacoma** [jprjr/sockexec#11]
After this change https://github.com/jprjr/sockexec/commit/63f79399b469152e68b1eb24cc490e5dbb63f7ff
My CI job fails with error:
```
src/common.h:21:30: fatal error: skalibs/posixplz.h: No such file or directory
 #include <skalibs/posixplz.h>
```

my env:
   alpine:3.7 

Dockerfile: https://github.com/metacoma/silverkey-web/blob/master/Dockerfile

**Comment by jprjr** [comment]
Sorry about that! Looks like Alpine uses a version of skalibs that doesn't include that header.

I've released version 3.0.0 of sockexec -- this includes a configure script so at build-time, I can determine if I should include `skalibs/posixplz.h` vs `skalibs/environ.h`

I just compiled + installed this on Alpine 3.7 with:

```
apk add --no-cache git musl-dev gcc make skalibs-dev
git clone https://github.com/jprjr/sockexec /tmp/sockexec
./configure --prefix=/usr --bindir=/usr/local/bin
make install
```

**Comment by metacoma** [comment]
@jprjr thanks for fast fix!
